plugins/module_utils/foreman_helper.py

```diff
@@ -1778,20 +1778,26 @@ def build_fqn(name, parent=None):
 
 # Helper for puppetclasses
 def ensure_puppetclasses(module, entity_type, entity, expected_puppetclasses=None):
-    puppetclasses_resource = '{0}_classes'.format(entity_type)
     if expected_puppetclasses:
```

Comment:
I think, this function should do nothing, if `puppetclasses` was not specified.

Reply:
I tend to agree, even tho this is cosmetical, the old could didn't do *much* without them.